css/elements/button.css

```diff
@@ -198,3 +198,13 @@
 .btn.is-success.is-inverted:hover {
     background-color: var(--white-grey);
 }
+
+/* ---- Group Of Buttons ---- */
+
+.buttons .btn {
+    margin-bottom: var(--size-5);
+}
+
+.buttons .btn:not(:last-child) {
+    margin-right: var(--size-5);
+}
```

Comment:
Add a new line at the EOF

Reply:
Its already there, else it would show huge circle dot with a line in it.